recipes/espnet/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "espnet" %}
+{% set version = "202205" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/espnet-{{ version }}.tar.gz
+  sha256: c7bfc2e76a25bf67b82de7165e201861c8ca061e413715583bf84cb963576197
+
+build:
+  skip: true  # [win]
```

Comment:
What is the reason that this package is not built as noarch?

Reply:
See PyTorch comment above.

Reply:
Please see comment above about missing dependencies.